subsys/nrf_security/src/drivers/Kconfig.psa_accel

```diff
@@ -0,0 +1,821 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config PSA_ACCEL_AES128_CBC_MAC
+	bool
+
+config PSA_ACCEL_AES128_CBC_NO_PADDING
+	bool
+
+config PSA_ACCEL_AES128_CBC_PKCS7
+	bool
+
+config PSA_ACCEL_AES128_CCM
+	bool
+
+config PSA_ACCEL_AES128_CCM_STAR_NO_TAG
+	bool
+
+config PSA_ACCEL_AES128_CFB
+	bool
+
+config PSA_ACCEL_AES128_CMAC
+	bool
+
+config PSA_ACCEL_AES128_CTR
+	bool
+
+config PSA_ACCEL_AES128_ECB_NO_PADDING
+	bool
+
+config PSA_ACCEL_AES128_GCM
+	bool
+
+config PSA_ACCEL_AES128_OFB
+	bool
+
+config PSA_ACCEL_AES128_XTS
+	bool
+
+config PSA_ACCEL_AES192_CBC_MAC
+	bool
+
+config PSA_ACCEL_AES192_CBC_NO_PADDING
+	bool
+
+config PSA_ACCEL_AES192_CBC_PKCS7
+	bool
+
+config PSA_ACCEL_AES192_CCM
+	bool
+
+config PSA_ACCEL_AES192_CCM_STAR_NO_TAG
+	bool
+
+config PSA_ACCEL_AES192_CFB
+	bool
+
+config PSA_ACCEL_AES192_CMAC
+	bool
+
+config PSA_ACCEL_AES192_CTR
+	bool
+
+config PSA_ACCEL_AES192_ECB_NO_PADDING
+	bool
+
+config PSA_ACCEL_AES192_GCM
+	bool
+
+config PSA_ACCEL_AES192_OFB
+	bool
+
+config PSA_ACCEL_AES192_XTS
+	bool
+
+config PSA_ACCEL_AES256_CBC_MAC
+	bool
+
+config PSA_ACCEL_AES256_CBC_NO_PADDING
+	bool
+
+config PSA_ACCEL_AES256_CBC_PKCS7
+	bool
+
+config PSA_ACCEL_AES256_CCM
+	bool
+
+config PSA_ACCEL_AES256_CCM_STAR_NO_TAG
+	bool
+
+config PSA_ACCEL_AES256_CFB
+	bool
+
+config PSA_ACCEL_AES256_CMAC
+	bool
+
+config PSA_ACCEL_AES256_CTR
+	bool
+
+config PSA_ACCEL_AES256_ECB_NO_PADDING
+	bool
+
+config PSA_ACCEL_AES256_GCM
+	bool
+
+config PSA_ACCEL_AES256_OFB
+	bool
+
+config PSA_ACCEL_AES256_XTS
+	bool
+
+config PSA_ACCEL_AES_CMAC_PRF_128
+	bool
+
+config PSA_ACCEL_ARIA
+	bool
+
+config PSA_ACCEL_CAMELLIA
+	bool
+
+config PSA_ACCEL_CHACHA20
+	bool
+
+config PSA_ACCEL_CHACHA20_POLY1305
+	bool
+
+config PSA_ACCEL_DES
+	bool
+
+config PSA_ACCEL_ECDH_P224
+	bool
+
+config PSA_ACCEL_ECDH_P256
+	bool
+
+config PSA_ACCEL_ECDH_P384
+	bool
+
+config PSA_ACCEL_ECDH_X25519
+	bool
+
+config PSA_ACCEL_ECDH_X448
+	bool
+
+config PSA_ACCEL_ECDSA_ED25519
+	bool
+
+config PSA_ACCEL_ECDSA_ED448
+	bool
+
+config PSA_ACCEL_ECDSA_P224_SHA1
```

Comment:
Please follow the naming convention for PSA Crypto for hash algorithms using underscore between hash type and size

E.g: https://armmbed.github.io/mbed-crypto/html/api/ops/hashes.html?highlight=sha_1#c.PSA_ALG_SHA_1